### DISCUSSION TITLE
Alternate language support?

### DISCUSSION THREAD

**Discussion by cc8080** [weather-gov/api#476] [Q&A] [answered]
I wanted to ask if there will be support for alternate languages for the forecast and alerts.

Currently, you can get a Spanish forecast using, for example:

`https://www.nws.noaa.gov/wtf/MapClick.php?lat=37.498545000000036&lon=-122.250475&lg=sp`

The  `lg` parameter specifies the language.  I don't see anything in the API spec that talks about a language selector.

I'd appreciate info on whether this is planned, or not. Thanks.

And, if anyone knows how to get the Spanish version via the DWML data source, that would be handy, too.

**Comment by StephenClouse** [comment, marked as answer]
It is planned to support Spanish forecast text and alerts in API, but not yet implemented.

**Reply by cc8080** [reply]
Thanks!

**Comment by akirmse** [comment]
It's been a few years and I just ran into the same thing.  Is support coming soon?

**Comment by mnk98** [comment]
Any Update after so many year on this?

**Reply by sullynole** [reply]
I am not sure whether there is anything api-specific, but there is a translation project where you can leave this feedback if it doesnt yet have what you're looking for:  https://www.weather.gov/translate
